api/models/orderDao.js

```diff
@@ -0,0 +1,62 @@
+const { appDataSource } = require("../database/database");
+
+
+async function createOrderTable(userId, totalCost) {
+    try {
+			const orderTable = await appDataSource.query(
+				`
+					INSERT INTO orders 
+						(user_id, total_cost) 
+					VALUES (?, ?)
+    		`, [ userId, totalCost ]);
+
+    return orderTable;
```

Comment:
```suggestion
			return appDataSource.query(
				`
					INSERT INTO orders 
						(user_id, total_cost) 
					VALUES (?, ?)
    		`, [ userId, totalCost ]);
```
불필요한 변수 선언 없이 바로 return 해주시면 될 것 같습니다.